lms/models/group_info.py

```diff
@@ -78,3 +80,36 @@ class GroupInfo(BASE):  # pylint:disable=too-few-public-methods
 
     #: The value of the custom_canvas_course_id param this group was last launched with.
     custom_canvas_course_id = sa.Column(sa.UnicodeText())
+
+    #: A dict of info about this group.
+    info = sa.Column(MutableDict.as_mutable(JSONB))
+
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault("info", {})
+        kwargs["info"].setdefault("instructors", [])
+        super().__init__(*args, **kwargs)
+
+    @property
+    def instructors(self):
+        return self.info["instructors"]
```

Comment:
So by the looks of it, to interact with a JSON field you just use it like data structure. That's pretty neat.

Reply:
Yeah, you just get a Python dict or list (I didn't check exactly what type of object it really is)